azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/util/AzureCliUtils.java

```diff
@@ -78,7 +82,14 @@ public static void ensureMinimumCliVersion() {
     @Nonnull
     public static String executeAzureCli(@Nonnull String command) {
         try {
-            return CommandUtils.exec(command);
+            final InetSocketAddress proxy = Azure.az().config().getHttpProxy();
```

Comment:
Extract a method like https://github.com/microsoft/azure-tools-for-java/pull/5529/files#diff-f2251308a586ba84c6f22986178e4f25434cb5058e179440238625b36c2c1291R157?

Besides, shall we merge the two utils?

Reply:
refactor will not be proper in this PR